docs/modules/account-setup-and-admin/pages/how_to_open_and_submit_a_support_ticket.adoc

```diff
@@ -0,0 +1,24 @@
+# How to Open and Submit a Support Ticket
+
+If you run into any issues, please reach out to our Support team by opening a support ticket from the portal or support.kobiton.com.
+
+However, we do encourage you to check out our documentation or look through our FAQs - you may just find the answers you need. 
+
+Submit a ticket from the Kobiton portal.
+To open a support ticket from the Kobiton portal: 
+
+Click on the Support icon, located at the top right of the portal, which will redirect you to support.kobiton.com
+[support-icon-on-kobi-portal.png]()
+
+
+Click on Submit a request
```

Comment:
@frankwmoyer - You should use a hyperlink here.

For example, the text should read:
> The best way is to submit a request.